README.md

```diff
@@ -73,7 +73,7 @@ More detail on some key features and implementations can be found below:
 - [Kubernetes and Knative integration](docs/kubernetes.md)- deploying Faasm as part of a full serverless platform.
 - [Bare metal/ VM deployment](docs/bare_metal.md) - deploying Faasm on bare metal or VMs as a stand-alone system.
 - [API](docs/api.md) - invoking and managing functions and state through Faasm's HTTP API.
-- [MPI](docs/mpi.md) and [OpenMP](docs/openmp.md) - executing existing MPI and OpenMP applications in Faasm.
```

Comment:
Is there value in linking the OpenMp docs from the cpp repo here? Maybe worth for the future to have accessible documentation about MPI and OpenMP in Faasm.

Reply:
If the OpenMP docs said anything about the Faasm implementation of OpenMP then there would be, however, those docs are purely about debugging issues with compilation, which is only relevant to the `cpp` repo.